modules/installing-rhv-config-yaml.adoc

```diff
@@ -4,66 +4,130 @@
 // * installing/installing_rhv/installing-rhv-default.adoc
 
 [id="installing-rhv-installation-configuration-parameters_{context}"]
-= Installation configuration parameters for {rh-virtualization}
+= Example `install-config.yaml` files for {rh-virtualization-first}
 
-Before you deploy an {product-title} cluster, you provide parameter values to describe your account on the cloud platform that hosts your cluster and optionally customize your cluster's platform. When you create the `install-config.yaml` installation configuration file, you provide values for the required parameters through the command line. If you customize your cluster, you can modify the `install-config.yaml` file to provide more details about the platform.
+You can customize the {product-title} cluster the installation program creates by changing the parameters and parameter values in the `install-config.yaml` file.
 
-The following example is specific to installing {product-title} on {rh-virtualization}. It uses numbered callouts to show which parameter values you can edit. Do not modify the parameters values without callouts.
+The following example is specific to installing {product-title} on {rh-virtualization}.
 
-This file is located in the `<installation directory>` you specified when you ran the following command.
+This file is located in the `<installation_directory>` you specified when you ran the following command.
 [source,terminal]
 ----
 $ sudo ./openshift-install create install-config --dir=<installation_directory>
```

Comment:
Since [this bug](https://bugzilla.redhat.com/show_bug.cgi?id=1820492) has been fixed, it should not be necessary any more to run the installation with root privileges.

Reply:
Please disregard, I noticed that this is being taken care of as part of https://github.com/openshift/openshift-docs/pull/25091